packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -320,6 +323,35 @@ class ConfigScreenComponent extends BaseScreenComponent {
 		return 0;
 	}
 
+	private handleBackButtonPress = (): boolean => {
+		const goBack = async () => {
```

Comment:
Not async?

Reply:
I was missing an `await` before the `BackButtonService.back` — it should be `async`.